pandas/_libs/window/indexers.pyx

```diff
@@ -138,7 +138,12 @@ def calculate_variable_window_bounds(
                         break
             # end bound is previous end
             # or current index
-            elif (index[end[i - 1]] - end_bound) * index_growth_sign <= 0:
+            elif index[end[i - 1]] == end_bound:
```

Comment:
Could you make a similar test for `VariableOffsetWindowIndexer.get_window_bounds` and include a test?

Reply:
Done